zkstack_cli/crates/zkstack/src/commands/ecosystem/args/init.rs

```diff
@@ -106,6 +106,8 @@ pub struct EcosystemInitArgs {
 impl EcosystemInitArgs {
     pub fn get_genesis_args(&self) -> GenesisArgs {
         GenesisArgs {
+            mode: ExecutionMode::Release,
+            tag: None,
```

Comment:
is it possible to unify it as a enum?

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3314/commits/f990a0ae25fe8896df91838f1f06bcf4c622c827